bootstrap/secret/jwtprovider.go

```diff
@@ -14,23 +14,15 @@ import (
 )
 
 type jwtSecretProvider struct {
-	secretProvider interfaces.SecretProviderExt
-	roundTripper_a http.RoundTripper
+	secretProvider      interfaces.SecretProviderExt
+	defaultRoundTripper http.RoundTripper
 }
 
 func NewJWTSecretProvider(secretProvider interfaces.SecretProviderExt) clientinterfaces.AuthenticationInjector {
 	return &jwtSecretProvider{
 		secretProvider: secretProvider,
 	}
 }
-func NewJWTSecretProviderWithRT(secretProvider interfaces.SecretProviderExt, roundTripper_b http.RoundTripper) clientinterfaces.AuthenticationInjector {
```

Comment:
will this removeal break anything?

Reply:
[I searched acrosss the whole org with github for the function but found nothing ](https://github.com/search?q=org%3Aedgexfoundry%20NewJWTSecretProviderWithRT&type=code)using it. I was surprised to see an unused function so I removed it. tbh it sort of looked like it was the nascent attempt at providing fallback support.

Reply:
ok, thanks